scripts/tests/z-wave-stack-binaries-test.sh

```diff
@@ -255,43 +285,39 @@ controller_cli_()
 }
 
 
-node_()
-{
-    node_app=$(realpath "${z_wave_stack_binaries_bin_dir}/ZW_zwave_soc_switch_on_off_"*"_REALTIME"*".elf" | head -n1)
-    [ "$debug" = "" ] || file -E "${node_app}"
-    ${node_app} --pty
-}
-
-
 node_cli_()
 {
-    while [ ! -e "${node_log}" ] ; do sleep 1; done
-    screen -S "$name" -p 2 -t "node" -X stuff "$@^M"
+    local node
+    [ -z $1 ] || { node="$1" && shift ; }
+    [ ! -z $node ] || node="soc_switch_on_off"
+    local node_log="$node${log_suffix}"
+    until [ -e "${node_log}" ] ; do sleep 1; done
+    screen -S "$name" -p "${node}" -X stuff "$@^M"
     sleep 1
     case $1 in
         h)
-            echo "node: Should display help"
+            echo "node: ${node}: Should display help"
             ;;
         d)
             DSK=$(grep 'DSK: ' "${node_log}" \
```

Comment:
piggyback on this PR?

```suggestion
            local DSK=$(grep 'DSK: ' "${node_log}" \
```